foodnet/membership/fixtures/initial_data.json

```diff
@@ -0,0 +1,27 @@
+[
+{
+    "model": "membership.membercategory",
+    "pk": 1,
+    "fields": {
+        "name": "houshold"
+    }
+},
+{
+    "model": "membership.membercategory",
+    "pk": 2,
+    "fields": {
+        "name": "school"
+    }
+},
+{
+    "model": "membership.division",
+    "pk": 1,
+    "fields": {
+        "allow_webmembers": true,
+        "category": "food",
+        "name": "default",
+        "contact": "admin@localhost",
+        "shortname": "def"
+    }
+}
+]
```

Comment:
Fixtures are a pain in the ass to maintain. I'd just go with a management command to create the data, or just use model mommy or similar if you want the data for tests.